client/src/components/MovieNotFound/style.scss

```diff
@@ -0,0 +1,14 @@
+@import "../../styles/master";
+.movie-not-found {
+  p {
+    color: #fff;
+    a {
+      color: $color-negative;
+      text-decoration: none;
+
+      &:hover {
```

Comment:
Hover's style must be wrapper by a media-query, because many mobile devices emulate hovering when the user performs an inconvenient long tap

```css
@media (hover: hover) {
  a:hover {
    background: yellow;
  }
}
```

Read more https://developer.mozilla.org/en-US/docs/Web/CSS/@media/hover